draft-jhoyla-req-mtls-flag.md

```diff
@@ -114,15 +143,27 @@ terminate the connection with a fatal `illegal_parameter` alert.
 
 # Security Considerations
```

Comment:
would it be relevant to point to security consideration section of TLS flags ? https://datatracker.ietf.org/doc/html/draft-ietf-tls-tlsflags-15#name-security-considerations

Reply:
would it make sense to update the wording ofn tls flag draft then?

Reply:
Although now I think on it, maybe we should just say "ignore" because otherwise we're giving servers a way to detect whether a client understands this draft. A client that ignores the incorrect flag doesn't, one that aborts does.